fe/fe-core/src/main/java/com/starrocks/consistency/ConsistencyChecker.java

```diff
@@ -95,6 +96,8 @@ public class ConsistencyChecker extends FrontendDaemon {
     private int endTime;
     private long lastTabletMetaCheckTime = 0;
 
+    private final Map<Long, Integer> creatingTableIds = new ConcurrentHashMap<>();
```

Comment:
```suggestion
    private final Map<Long, Integer> creatingTableCounters = new ConcurrentHashMap<>();
```
also you need to explain this structure, it's kind of hard to understand.

Reply:
done